test/Interop/Cxx/templates/dependent-types-module-interface.swift

```diff
@@ -1,12 +1,10 @@
 // RUN: %target-swift-ide-test -print-module -module-to-print=DependentTypes -I %S/Inputs -source-filename=x -enable-experimental-cxx-interop | %FileCheck %s
 
 // CHECK: func differentDependentArgAndRet<T, U>(_ a: Any, T: T.Type, U: U.Type) -> Any
-// CHECK: func dependantReturnTypeInffered<T>(_ a: T) -> Any
+// CHECK: func dependantReturnTypeInferred<T>(_ a: T) -> Any
 // CHECK: func dependantReturnTypeSameAsArg<T>(_ a: Any, T: T.Type) -> Any
+// CHECK: func complexDependantReturnTypeInferred<T>(_ a: T) -> Any
 // CHECK: func multipleArgs<T>(_ a: Any, _ b: T, _ c: Int32) -> Any
 // CHECK: func multipleDependentArgsInferred<T, U>(_ a: Any, _ b: Any, _ c: T, _ d: U) -> Any
 // CHECK: func multipleDependentArgs<T, U>(_ a: Any, _ b: Any, T: T.Type, U: U.Type) -> Any
-// CHECK: func refToDependent<T>(_ a: inout T) -> Any
-// We don't support references to dependent types (rdar://89034440).
-// CHECK-NOT: dependentRef
```

Comment:
Can you turn these into CHECK:s for these two functions?

Reply:
Please also try using them in the executable test.

Reply:
Currently still not imported